recipes/dabl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "dabl" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 54cfaefaa9e32f740ee3eeec9c3cf114cc8a1128dde1c1adf363441321e88f28
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib
+    - numpy
+    - pandas
+    - scikit-learn
+    - scipy
+    - seaborn
+
+test:
+  imports:
+    - dabl
+
+about:
+  home: https://github.com/dabl/dabl
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
The license isn't packaged in the sdist so, for now, you need to include a copy of it alongside the recipe.

Reply:
***xref:*** https://github.com/dabl/dabl/pull/253